### DISCUSSION TITLE
feat-maker-academy/academy/category1/lesson1/

### DISCUSSION THREAD

**Discussion by giscus** [tolocar-project/website#44] [Announcements]
# feat-maker-academy/academy/category1/lesson1/

The Tolocar project sends mobile makerspaces into Ukraine to help, empower, equip and train local communities through open technology, knowledge and innovation.

https://dev.tolocar.org/feat-maker-academy/academy/category1/lesson1/

**Comment by pReya** [comment]
Test Hallo
